codegen.yml

```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: "https://api-gateway.litentry.io/graphql"
+schema: "https://graph.litentry.io/graphql"
```

Comment:
Shouldn't we also change it in the `LitentryApiContext`?

Reply:
yep